docs/content/direct/example-scenarios.md

```diff
@@ -117,16 +117,13 @@ kubectl --context "$wds_context" delete bindingpolicies nginx-bpolicy
 
 ## Scenario 2 - Out-of-tree workload
 
-This scenario is like the previous one but involves a workload whose
-kind of objects is not built into Kubernetes. Instead, the workload
-object kind is defined by a `CustomResourceDefinition` object. While
-KubeStellar can handle the case where the CRD is part of the workload,
-this example concerns the case where the CRD is established in the
-WECs by some other means.
+This scenario demonstrates deploying custom resources (CRDs) and highlights **KubeStellar's authorization model**. Unlike standard Kubernetes resources, custom resources require explicit authorization expansion for the OCM Agent.
```

Comment:
The deploying of the custom resource _definitions_ is not the point of this example, it is a supporting activity. The point of this example is distributing a kind of object that is not built into Kubernetes.